clippy_lints/src/eq_op.rs

```diff
@@ -236,7 +236,7 @@ impl<'tcx> LateLintPass<'tcx> for EqOp {
                                     rsnip,
                                     Applicability::MaybeIncorrect, // FIXME #2597
                                 );
-                            })
+                            });
```

Comment:
Seeing the churn in Clippy, I would say that this is a `restriction` lint rather than a `pedantic` lint.

Reply:
Alright, that would mean reverting all the clippy changes I guess.

Reply:
Yep, that's why you put fixes like this in separate commits. 😉 It's also easier to review or reviewing it with those dogfood fixes in relevant commits is just painful.